default_translations/de/messages.de.yml

```diff
@@ -35,12 +35,14 @@ form:
   voucher-redeemed-on: Eingelöst am
   oclock-by: Uhr von
   actual-password: Aktuelles Passwort
+  new-custom-alias: Neues spezifisches Alias
```

Comment:
I would name it only `Neue Weiterleitung` or `Neues Alias`

Reply:
lets discuss this at a later point, when we have some feedback on the aliases

Reply:
Why not change 'alias' to 'alias address' (en) and 'Alias' to 'Alias Adresse' (de) now? I think, that already helps a lot in understanding that it's about *mail addresses*.

Reply:
Okay, i changed it to `alias address`